Makefile

```diff
@@ -595,6 +595,12 @@ test-message: test-build
 test-wpt: all
 	$(PYTHON) tools/test.py $(PARALLEL_ARGS) wpt
 
+.PHONY: test-wpt-report
+test-wpt-report:
+	$(RM) -r out/wpt
+	mkdir -p out/wpt
+	WPTREPORT=1 $(PYTHON) tools/test.py --shell $(NODE) $(PARALLEL_ARGS) wpt
```

Comment:
Mostly the same as `test-wpt` but with --shell to support globally installed Node.js version if available, otherwise the one built from source. Also and `env.WPTREPORT` signal for the WPTRunner to create one.